modules/context/org.go

```diff
@@ -145,6 +148,11 @@ func HandleOrgAssignment(ctx *Context, args ...bool) {
 				ctx.ServerError("IsOrgMember", err)
 				return
 			}
+			ctx.Org.IsAdmin, err = org.IsOrgAdmin(ctx, ctx.Doer.ID)
```

Comment:
Shouldn't we only execute this check if the user is actually a member?

Reply:
Did some changes in https://github.com/go-gitea/gitea/pull/28838/commits/84bab561df3838828cc52cfa103f808209f0a28d